web/src/app/modules/overview/services/websocket/websocket.service.ts

```diff
@@ -182,3 +169,26 @@ export class WebsocketService implements BackendService {
     }
   }
 }
+
+const websocketURI = () => {
+  const loc = window.location;
+  let newURI = '';
+  if (loc.protocol === 'https:') {
+    newURI = 'wss:';
+  } else {
+    newURI = 'ws:';
+  }
+  newURI += '//' + loc.hostname;
+
+  let port = loc.port;
+  if (loc.port === '4200') {
+    // If port is 4200, dev mode is likely active. Replace port with
```

Comment:
`console.log` here about what we are doing

Reply:
removed this commit. @wwitzel3  please review!